webui/tests/tests/StepImplementation.ts

```diff
@@ -46,319 +46,208 @@ const BASE_PATH = process.env.PUBLIC_URL || '/det';
 const BASE_URL = `${HOST}${BASE_PATH}`;
 const SCREENCAST = process.env.SCREENCAST === 'true';
 
-function handleError(e) {
-  console.error(e);
-}
-
 export default class StepImplementation {
   @BeforeSuite()
   public async beforeSuite() {
```

Comment:
changes here should just be removing trycatch blocks but the diff is bigger since we're removing a level of indentation